spark-clickhouse-connector/src/main/scala/io/clickhouse/spark/connector/ClickhouseRDD.scala

```diff
@@ -93,6 +93,20 @@ class ClickhouseRDD
       clickhousePartitioner = new SimpleClickhousePartitioner(connector)
     )
   }
+
+  /**
+    * Dated partitioning strategy: Dated partition for each shard.
+    * * @param startDt
+    * * @param endDt
+    * * @param rangeType
+    * * @param partitionKeyName
+    * * @return
+    */
+  def withoutDatedPartitioning(startDt: DateTime, endDt: DateTime, rangeType: RangeType, partitionKeyName: String): Self = {
```

Comment:
Why you need this method? What is the difference between this method and the "withPeriod" method? Also very strange naming for this method.